### PR TITLE
Rainbow

### PR DESCRIPTION
**Pull request by nicm42**
### New Stylesheet Submission:

- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other files using the same name as yours?
- [x] Have you filled in at least the required data for `title`, `author`, and `stylesheet`?
- [x] Is the value of `stylesheet` a full URL that is publicly accessible and renders an unminified, compiled CSS file?
- [x] Do your styles pass proper contrast for accessibility?
- [x] Have you included `@media (prefers-reduced-motion: reduce)` to remove animations on `*,*::before,*::after`?
- [x] Did you style the `.skip-link`?
- [x] Have you checked on a real mobile device to ensure your responsive design doesn't cause overlaps or overflow scrolling?

### CODE REVIEW

File: src/_data/styles/rainbow.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  // Required
```
Comment (5t3ph) [inline src/_data/styles/rainbow.json:2]:
```suggestion
```

File: src/_data/styles/rainbow.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  "title": "Rainbow",
+  "author": "Nic Mayer",
+  "stylesheet": "https://raw.githubusercontent.com/nicm42/Style-Stage-Rainbow/master/style.css",
+  // Optional
```
Comment (5t3ph) [inline src/_data/styles/rainbow.json:6]:
```suggestion
```

**Review by 5t3ph — CHANGES_REQUESTED** [review]

**Comment by 5t3ph** [comment]
Hi @nicm42 - one small thing!

The `:hover/:focus` on the links in the header needs to either be juuuust a little bit darker green, or bump the font-size over 18.5px:
image

**Comment by nicm42** [comment]
Hi. I've fixed that text to be at least 20px.

**Comment by 5t3ph** [comment]
Hi @nicm42 - looks like you need to merge your change into your master branch to wrap up this change, or update the URL here to point to your `v2` branch file.

**Comment by nicm42** [comment]
Sorry, I was convinced when I looked at it earlier that I'd done it from the branch. I've updated the master branch now.

**Review by 5t3ph — APPROVED** [review]
@nicm42 change is good, publishing!

Thank you for participating! If you haven't already, consider subscribing for updates to learn about new styles and features: https://stylestage.dev/subscribe/

**Pull request merged**